src/components/line.js

```diff
@@ -5,7 +5,7 @@ module.exports.Component = registerComponent('line', {
   schema: {
     start: {type: 'vec3', default: {x: 0, y: 0, z: 0}},
     end: {type: 'vec3', default: {x: 0, y: 0, z: 0}},
-    color: {type: 'color', default: '#74BEC1'},
+    color: {type: 'color', default: '#fff'},
```

Comment:
The default color is used by the raycaster and laser controls, I wouldn't make it white since it'd be hard to see.